_posts/2021-02-18-how-to-build-knapsack-pro-api-client-from-scratch-in-any-programming-language.md

```diff
@@ -116,6 +116,8 @@ Users can define those environment variables in their CI server settings to cont
   - If no value is detected then a `git rev-parse --abbrev-ref HEAD` command should be run to determine the branch name.
   - If `git` is not installed then raise an error. Please see the [source code of `@knapsack-pro/core`](https://github.com/KnapsackPro/knapsack-pro-core-js/blob/0f44c6a3daa369cd4353e315abbf5539295289ea/src/config/knapsack-pro-env.config.ts#L147,L184).
 
+- `KNAPSACK_PRO_USER_SEAT` - SHA-256 hash of the user name that started the CI build. Often it's the same person that created the git commit.
```

Comment:
I would rephrase the 2nd sentence to "It is usually the same person that made the git commit."

Reply:
same [here](https://github.com/KnapsackPro/docs.knapsackpro.com/pull/111/files#diff-80fc479f22c287ec8a449e40bce654d92d1a54bee37530f31fcd237f1012a2cbR229)